schemas/sds/1.2/scap-source-data-stream_1.2.xsd

```diff
@@ -9,7 +9,7 @@
     </xs:appinfo>
   </xs:annotation>
   <xs:import namespace="http://checklists.nist.gov/xccdf/1.2" schemaLocation="../../xccdf/1.2/xccdf_1.2.xsd"/>
-  <xs:import namespace="http://oval.mitre.org/XMLSchema/oval-definitions-5" schemaLocation="../../oval/5.10/oval-definitions-schema.xsd"/>
+  <xs:import namespace="http://oval.mitre.org/XMLSchema/oval-definitions-5" schemaLocation="../../oval/5.11/oval-definitions-schema.xsd"/>
```

Comment:
Please include an XML comment around this change to make it more obvious that we changed the schema.

This is the official schema. It imports 5.10.

http://scap.nist.gov/schema/scap/1.2/scap-source-data-stream_1.2.xsd